docs/architecture/ADR-005-QGB_posting_confirms_in_reserved_namespace.md

```diff
@@ -0,0 +1,97 @@
+# ADR 5: Reduce QGB state
+
+## Changelog
+
+- {date}: {changelog}
+
+## Context
+
+Currently, the QGB is using the state to save all the attestations and confirms it needs.
```

Comment:
> the state

[optional] it may help non-familiar readers (i.e. me) to clarify what this state is. Is it specifically state stored within the QGB module which would end up bloating the celestia-app state?

Reply:
```suggestion
Currently, the QGB module is using the Celestia-app state to save all the attestations and confirms it needs.
```
?

Reply:
check now please?